ddterm.sh.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2025 thenameisluk
```

Comment:
By the way, a separate license file is necessary only if the "main" file doesn't allow comments. Here you could include the same info in the beginning of `.sh` as a comment, and `.license` won't be necessary. In the main repository I also embed licensing info into `.svg` files (because svg is just xml) to reduce the file count in the package.

But it's just FYI, no need to fix.